rules/private/intellij_modules_xml.bzl

```diff
@@ -30,6 +30,26 @@ def _impl(ctx):
         ],
         progress_message="Generating intellij modules.xml file: %s" % ctx.outputs.modules_xml_file.path)
 
+    shell_script_content = \
+"""#!/bin/bash -e
```

Comment:
leaving obvious duplication for now, will collapse it at example 2 or 3, or failing that, some cleanup phase in the future.